weeklymission2/src/Components/Header.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import logo from '../images/logo.svg';
+
+export function Header({ profile, folder }) {
+  return (
+    <>
+      <div className="headerArea">
+        <nav>
+          <div className="navBar">
+            <a href="index.html"
+              ><img src={logo} alt="logo" className="logo"
+            /></a>
+            {profile ?
+              (<div className="profile"><img src={profile.image} alt="userProfileImg" className="profileImg"></img>
+              <span className="userEmail">{profile.email}</span></div>) :
+              (<a href="../Pages/signIn.html" className="btn btnS">로그인</a>)
```

Comment:
디폴트 객체는 줬지만 프로퍼티는 디폴트가 `null`이죠? 데이터를 받아오기 전까지는 해당 프로퍼티는 null이니까 
여기에 디폴트 프로퍼티를 줘야합니다.

```suggestion
              (<div className="profile"><img src={profile.image || ""} alt="userProfileImg" className="profileImg"></img>
              <span className="userEmail">{profile.email || "No Email"}</span></div>) :
              (<a href="../Pages/signIn.html" className="btn btnS">로그인</a>)
```

Reply:
삼항연산자의 조건으로 profile.email을 넣어서 profile.email값은 항상 true라 "No Email"은 안넣어도 된다고 생각하는데 제 생각이 맞을까요??

Reply:
> 삼항연산자의 조건으로 profile.email을 넣어서 profile.email값은 항상 true라 "No Email"은 안넣어도 된다고 생각하는데 제 생각이 맞을까요??

그 말도 맞는 말이네요 !! . email이 있으니까 디폴트값을 안줘도 됩니다. !! ㅎㅎ 
좀 더 정확하게 하려면 email이 "" 과 같은 의미없는 스트링일때 디폴트 값을 줄 수 있도록 할 수 있겠네요 ㅎㅎ